tests/common/test_version.py

```diff
@@ -57,6 +57,13 @@ def default_system_exception():
     raise Exception
 
 
+def is_platform_supported():
```

Comment:
NIT: rename to `is_platform_dist_supported()`. `is_platform_supported` sounded more like if the platform supports it